index.js

```diff
@@ -0,0 +1,21 @@
+const express=require('express')
+const app=express();
+const morgan=require("morgan");
+const postRoutes=require("./routes/post");
+const date = require('date-and-time')
+const swaggerUI=require('swagger-ui-express'),
+    swaggerDocument=require('./swagger.json');
+
+const yaml = require('yamljs');
+const fs   = yaml.load('./swagger.yaml');
```

Comment:
Use proper naming conventions.
For example: `const swaggerYaml = yaml.load('./swagger.yaml');`